docs/getting-started.md

```diff
@@ -41,24 +41,37 @@ Antrea deployment manifest:
 
 ## Theia Installation
 
+In Theia v0.1.0, we introduce the Flow Aggregator Helm Chart to Theia
+From Theia v0.2.0 and Antrea v1.8.0, the Flow Aggregator Helm Chart is moved from Theia to Antrea.
+
 To enable both [Grafana Flow Collector](network-flow-visibility.md) and
-[NetworkPolicy Recommendation](networkpolicy-recommendation.md), please install
-Flow Aggregator and Theia by runnning the following commands:
+[NetworkPolicy Recommendation](networkpolicy-recommendation.md), please follow
+the steps below:
 
-```bash
-git clone https://github.com/antrea-io/theia.git
-helm install flow-aggregator theia/build/charts/flow-aggregator --set clickHouse.enable=true,recordContents.podLabels=true -n flow-aggregator --create-namespace
-helm install theia theia/build/charts/theia --set sparkOperator.enable=true -n flow-visibility --create-namespace
-```
+1. Download the Flow Aggregator Helm Chart and the Theia Helm Chart.
 
-To enable only Grafana Flow Collector, please install Flow Aggregator and Theia
-by runnning the following commands:
+    For Theia v0.1.0, please clone the repository and checkout branch `release-0.1`.
+    The Helm charts are located under the folder `build/charts`.
 
-```bash
-git clone https://github.com/antrea-io/theia.git
-helm install flow-aggregator theia/build/charts/flow-aggregator --set clickHouse.enable=true -n flow-aggregator --create-namespace
-helm install theia theia/build/charts/theia -n flow-visibility --create-namespace
-```
+    From Theia v0.2.0 and Antrea v1.8.0, the Flow Aggregator Helm Chart is moved from
+    Theia repository to Antrea repository; and the Helm Charts are added to release
+    assets. Please download `theia-chart.tgz` from [Theia release assets](https://github.com/antrea-io/theia/releases)
+    and `flow-aggregator-chart.tgz` from [Antrea release assets](https://github.com/antrea-io/antrea/releases).
```

Comment:
Probably we also need to provide steps for unzipping these .tgz before installing

Reply:
Make sense. Added, thanks!